Games/Game

```diff
@@ -0,0 +1,459 @@
+import random
+
+randomNumber = random.randint(1, 15)
+##print(int(randomNumber))
+#""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+var = (int(randomNumber))
```

Comment:
If these `var...` variables are supposed to be used for specific purposes then name them accordingly.